cellpack/autopack/ingredient/Ingredient.py

```diff
@@ -3043,20 +3015,6 @@ def attempt_to_pack_at_grid_location(
                 moving,
```

Comment:
cellpack/autopack/ingredient/Ingredient.py Line 2546 `def get_rapid_nodes(self, close_indice, curentpt, removelast=False, prevpoint=None):`

Is this function still required after removing rapid?

PS: for some reason github won't let me leave comments at line 2546, so leaving it here for now.

Reply:
thanks, removed!